dango/dex/src/cron.rs

```diff
@@ -697,7 +666,7 @@ fn clear_orders_of_pair(
         }
 
         metrics::histogram!(
-            crate::metrics::LABEL_DURATION_HANDLE_FILLED,
+            crate::metrics::LABEL_DURATION_ORDER_FILLING,
```

Comment:
The "handle filled" step is merged into the "fill orders" step. @Rhaki the dashboard needs to be updated.